tensorflow/compiler/tf2tensorrt/utils/trt_logger.cc

```diff
@@ -44,14 +47,42 @@ void Logger::log(Severity severity, const char* msg) noexcept {
       LOG(FATAL) << name_ << " " << msg;
       break;
     }
+  }
+}
+
+void Logger::suppressLoggerMsgs(Severity severity) {
+  if (isValidSeverity(severity)) {
+    suppressedMsg_ |= 1 << (int)severity;
+  }
+}
+
+void Logger::unsuppressLoggerMsgs(Severity severity) {
+  if (isValidSeverity(severity)) {
+    suppressedMsg_ &= (-1) ^ (1 << (int)severity);
+  }
+}
+
+bool Logger::isValidSeverity(Severity severity, const char* msg) noexcept {
+  switch (severity) {
+    case Severity::kVERBOSE:
+    case Severity::kINFO:
+    case Severity::kWARNING:
+    case Severity::kERROR:
+    case Severity::kINTERNAL_ERROR:
+      return true;
     // This is useless for now. But would catch it in future if enum changes. It
     // is always good to have default case!
     default: {
-      LOG(FATAL) << name_ << "Got unknown severity level " << int(severity)
-                 << " from TensorRT: " << msg;
-      break;
+      std::stringstream ss;
+      ss << name_ << "Got unknown severity level " << int(severity);
+      if (msg) {
+        LOG(FATAL) << ss.str() << " from TensorRT: " << msg;
+      } else {
+        LOG(ERROR) << ss.str();
+      }
```

Comment:
Tensorflow has a new check that prevents new code with LOG(FATAL).
Can we just remove "default" to make this an exhaustive switch statement?

Reply:
DONE